config/rbac/role.yaml

```diff
@@ -11,67 +11,43 @@ rules:
   resources:
   - configmaps
   verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
+  - '*'
```

Comment:
ok, I see the point here, but let's make sure to have a good run later to actually REDUCE these permissions to the bare minimum, if possible. Now, probably for `configmaps` we cannot remove any permission, but in general our RBAC permissions will need some serious review later on.

Reply:
In general, all resource that the operator creates we can not reduce any permissions, because we need to create, update, get, watch, list and delete resources